lib/tapioca/runtime/reflection.rb

```diff
@@ -255,6 +255,35 @@ def child_module_for_parent_with_name(parent, name)
       def method_defined_by_forwardable_module?(method)
         method.source_location&.first == Object.const_source_location(:Forwardable)&.first
       end
+
+      sig { params(symbol: String).returns(String) }
+      def namespace_of(symbol)
```

Comment:
This doesn't need to be a separate method, we can fold it into how the other method operates.